apps/dashboard/src/fairness/__mock_data__/precomputedBinaryWithError.ts

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IFairnessData } from "@responsible-ai/core-ui";
+
+import { precomputedBinary } from "./precomputedBinary";
+
+Object.assign(precomputedBinary, {
```

Comment:
Am I understanding this correctly? We're assigning something new to `precomputedBinary` and then at the end of the file assign it to another variable that's exported. Why not use the exported one right away?

Reply:
Does this change precomputedBinary?

Reply:
I was trying to extend precomputedBinary, since what I'm exporting here is precomputedBinary with additional key-values

Reply:
Right, but does this change precomputedBinary which is expected not to have error bars (?). Just directly assign it to the variable at the end that you're exporting (?)
Or deep copy precomputedBinary and then assign to the copy, but I wouldn't change precomputedBinary.